karma.conf.coffee

```diff
@@ -3,7 +3,17 @@ packageJSON = require('./package.json')
 
 module.exports = (config) ->
 	karmaConfig.plugins.push(require('karma-chrome-launcher'))
-	karmaConfig.browsers = ['ChromeHeadless']
+	karmaConfig.browsers = ['ChromeHeadlessCustom']
+	karmaConfig.customLaunchers =
+		ChromeHeadlessCustom:
+			base: 'ChromeHeadless'
+			flags: [
+				'--no-sandbox'
+				'--no-proxy-server'
+				'--disable-setuid-sandbox'
+				'--disable-web-security'
```

Comment:
Do we definitely need this one? This shouldn't be required to get the browser to run, it'll change the runtime behaviour and make it more lax (which probably isn't what we want for testing)

Reply:
Already tried to keep the flags to the minimum but let my try removing this one as well now that the PR dust has settled.

Reply:
It ends-up that for the final alpine node 10 image (which is used in the npm publish job) that we ended up we only need the `--no-sandbox` (at least for now).
I will revisit these later while trying to fix the concourse test for node 6 & node 8.